assets/js/blocks/cart-checkout/checkout/inner-blocks/checkout-terms-block/test/edit.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * External dependencies
+ */
+import { render, findByRole, queryByText } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Edit } from '../edit';
+const blockSettingsMock = jest.requireMock( '@woocommerce/block-settings' );
+
+jest.mock( '@wordpress/block-editor', () => ( {
+	...jest.requireActual( '@wordpress/block-editor' ),
+	useBlockProps: jest.fn(),
+} ) );
+
+jest.mock( '@woocommerce/block-settings', () => ( {
+	...jest.requireActual( '@woocommerce/block-settings' ),
+	PRIVACY_URL: '/privacy-policy',
+	TERMS_URL: '/terms-and-conditions',
+} ) );
+
+describe( 'Edit', () => {
+	it( 'Renders a checkbox if the checkbox attribute is true', async () => {
+		const { container } = render(
+			<Edit
+				attributes={ {
+					text: 'I agree to the terms and conditions',
+					checkbox: true,
+				} }
+				setAttributes={ () => void 0 }
+			/>
+		);
+
+		expect( await findByRole( container, 'checkbox' ) ).toBeTruthy();
+	} );
+
+	it( 'Renders a notice if either the terms and conditions or privacy url attribute are unset', async () => {
+		blockSettingsMock.PRIVACY_URL = '';
+		blockSettingsMock.TERMS_URL = '';
+		const { container } = render(
+			<Edit
+				attributes={ {
+					text: 'I agree to the terms and conditions',
+					checkbox: true,
+				} }
+				setAttributes={ () => {} }
```

Comment:
nitpick: `() => void 0` here too. Should have mentioned there are a few occurances of this in my initial review, my bad!